pkgs/by-name/ru/rustdesk-flutter/package.nix

```
@@ -134,8 +139,12 @@ in flutter313.buildFlutterApplication rec {
     xdotool
   ];
 
-  postPatch = ''
+  prePatch = ''
     chmod -R +w ..
+  '';
+  patchFlags = "-p1 -d ..";
```

```suggestion
  patchFlags = [ "-p1" "-d" ".." ];
```
that should be an array